fuel/app/views/welcome/index.php

```diff
@@ -50,24 +50,24 @@
 			</div>
 			<div class="col-md-4">
 				<h2>Learn</h2>
-				<p>The best way to learn FuelPHP is reading through the <a href="http://docs.fuelphp.com">Documentation</a>.</p>
-				<p>Another good resource is the <a href="http://fuelphp.com/forums">Forums</a>.  They are fairly active, and you can usually get a response quickly.</p>
+				<p>The best way to learn FuelPHP is reading through the <a href="https://docs.fuelphp.com">Documentation</a>.</p>
+				<p>Another good resource is the <a href="https://fuelphp.com/forums">Forums</a>.  They are fairly active, and you can usually get a response quickly.</p>
```

Comment:
This one is still wrong?

Reply:
Pardon me. Could you tell me which one is still wrong?

The URL I changed in the last commit back then was `https://docs.fuelphp.com` to `https://fuelphp.com/docs`. After I delete the commit, it changed back to `https://docs.fuelphp.com`

The origin URL was `http://docs.fuelphp.com`

Reply:
It is not only docs, but also forums you changed. It should be `https://forums.fuelphp.com`.

Reply:
Hmm.. I don't remember changed the forum URL 🤔

But, OK. I will change the forum URL. Thank you for pointing out the problem

Reply:
Thanks, I'll merge when the change is applied.